NitroxPatcher/Patches/Dynamic/GoalManager_OnCompletedGoal_Patch.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using HarmonyLib;
+using NitroxClient.Communication.Abstract;
+using NitroxClient.GameLogic;
+using NitroxClient.MonoBehaviours;
+using NitroxModel.Core;
+using NitroxModel.Helper;
+using NitroxModel.Packets;
+
+namespace NitroxPatcher.Patches.Dynamic
+{
+    public class GoalManager_OnCompletedGoal_Patch : NitroxPatch, IDynamicPatch
+    {
+        private static readonly MethodInfo TARGET_METHOD = Reflect.Method(() => GoalManager.main.OnCompleteGoal(default(GoalType), default(string)));
+
+        public static void Postfix(GoalType goalType, string goalIdentifier)
```

Comment:
Just a note: You don't need to reference every argument from the original method if you don't need it in the patch.

Reply:
Removed in commit #4.